analyses/oncoprint-landscape/run-oncoprint.sh

```diff
@@ -0,0 +1,20 @@
+# Chante Bethell for CCDL 2019
+# Run 01-plot-oncoprint.R
+#
+# Usage: bash run-oncoprint.sh
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+script_directory="$(perl -e 'use File::Basename;
+  use Cwd "abs_path";
+  print dirname(abs_path(@ARGV[0]));' -- "$0")"
+cd "$script_directory" || exit
+
+mkdir driver-lists
+wget --directory-prefix=driver-lists -O driver-lists/brain-goi-list-long.txt https://github.com/marislab/create-pptc-pdx-oncoprints/raw/2c9ed2a2331bcef3003d6aa25a130485d76a3535/data/brain-goi-list-old.txt 
+wget --directory-prefix=driver-lists -O driver-lists/brain-goi-list-short.txt https://github.com/marislab/create-pptc-pdx-oncoprints/blob/2c9ed2a2331bcef3003d6aa25a130485d76a3535/data/brain-goi-list.txt
```

Comment:
I thought the `blob` -> `raw` was necessary and remembered to change the first one but perhaps not
```suggestion
wget --directory-prefix=driver-lists -O driver-lists/brain-goi-list-short.txt https://github.com/marislab/create-pptc-pdx-oncoprints/raw/2c9ed2a2331bcef3003d6aa25a130485d76a3535/data/brain-goi-list.txt
```